cg/apps/tb/dto/create_job_request.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+
+
+class CreateJobRequest(BaseModel):
+    slurm_id: str
+    job_type: str
```

Comment:
Let's restrict these to the same as defined in TB.

Reply:
The `slurm_id` exists and `job_type` will exist once I merge a PR 👍

Reply:
Added enum 👍